### ISSUE TITLE
The Vue 2 IMask Directive doesn't work anymore on CodeSandbox since version 6.2.0

### ISSUE THREAD

**Issue by tbl0605** [uNmAnNeR/imaskjs#638]
Hi,
please take example from [Vue IMask Plugin Directive Example (Vue 2)](https://github.com/uNmAnNeR/imaskjs/tree/master/packages/vue-imask#mask-directive-example) and use it in CodeSandbox, it won't work.

**To Reproduce**
I've copied the code here: [CodeSandbox](https://codesandbox.io/s/bug-vue2-imask-directive-tphgtd)

If you downgrade vue-imask to version 6.1.0 it will work again.

**Expected behavior**
The Vue 2 IMask Plugin Directive should work in CodeSandbox like it did with vue-imask <= 6.1.0

Thank you!

Thierry.

**Comment by tbl0605** [comment]
As an additional comment: my main concern is not only that the mentioned example no longer works, but that overall the Vue 2 IMask directive no longer works on CodeSandbox, my own CodeSandbox projects being impacted by this regression ;)

**Comment by tbl0605** [comment]
I found a workaround for this problem in CodeSandbox.
I simply extracted the Vue 2 IMask directive, rewrote it in CodeSandbox, and forced `isVue3` to be false (so without using vue-demi directly anymore).

See my workaround here: [CodeSandbox](https://codesandbox.io/s/bug-vue2-imask-directive-workaround-jgvoy7?file=/src/vue-imask-workaround.js)

At this point, I'm not sure if this is a Vue 2 IMask packaging issue, a vue-demi issue, or a CodeSandbox issue.

**Comment by uNmAnNeR** [comment]
seems like should be fixed by updating `vue-demi` to the latest version. check in the coming release

**Comment by tbl0605** [comment]
Hi @uNmAnNeR
Sadly no, I cannot see any difference with latest version of `vue-imask` and/or `vue-demi` :(

**Comment by uNmAnNeR** [comment]
i clone the project, changed version to 6.4.3, do `npm i` locally and it works as expected.
please reopen if you have more details

**Comment by uNmAnNeR** [comment]
@tbl0605 seems like the problem can be in using codesandbox. can you try it locally? for me it's working as expected.

**Comment by tbl0605** [comment]
@uNmAnNeR, it works locally, it seems only to be a codesandbox bug. When I add following code (in codesandbox):

```
import { isVue2, isVue3 } from "vue-demi";
```

`isVue2` will be `false` and `isVue3` will be `true` in codesandbox, even if I use Vue 2.7!

I tried to use `vue-demi-switch 2`and `vue-demi-fix` to fix that manually in codesandbox but it didn't change anything (maybe I did it wrong, but I found no codesandbox example how to use these commands).

Anyway, I'm ok with letting my bug report closed, I don't think it's worth trying to fix this bug, maybe one day I will have a better idea what's really happening in codesandbox ;)
I'm just curious to know if there are other Vue.js library (using vue-demi) having such kind of problems with codesandbox...
